contact-center/android-chat/app/src/main/java/com/example/vonage/chatsampleapp/chat/ChatClientManager.kt

```diff
@@ -99,34 +99,34 @@ class ChatClientManager(
     suspend fun getConversations(pageSize: Int = Constants.PAGE_SIZE,
                                  cursor: String? = null,
                                  order: PresentingOrder = Constants.DEFAULT_ORDER) : ConversationsPage =
-        client.getConversations(order, pageSize, cursor)
+        client.getConversations(GetConversationsParameters(order, pageSize, cursor))
```

Comment:
why did we go with Parameters over Options?

Reply:
you are late @NatoNathan  for this comment